memberboat/yaml/yaml.py

```diff
@@ -0,0 +1,17 @@
+import yaml
+
+
+def read(filepath):
+	file_data = []
+	try:
```

Comment:
Might want to refactor this to return data if not all of the files threw an exception.